tests/tests.go

```diff
@@ -228,6 +232,33 @@ func initKubemacpoolParams(rangeStart, rangeEnd string) error {
 	return nil
 }
 
+func getVmFailCleanupWaitTime() time.Duration {
+	managerDeployment := v1.Deployment{}
+	err := testClient.VirtClient.Get(context.TODO(), client.ObjectKey{Namespace: managerNamespace, Name: names.MANAGER_DEPLOYMENT}, &managerDeployment)
+	Expect(err).ToNot(HaveOccurred(), "Should successfully get manager's Deployment")
+	Expect(managerDeployment.Spec.Template.Spec.Containers).ToNot(BeEmpty(), "Manager's deployment should contain containers")
+
+	for _, container := range managerDeployment.Spec.Template.Spec.Containers {
+		if container.Name != deploymentContainerName {
```

Comment:
This `if` can be more straightforward by changing it to `==` and drop  `continue`
```
if container.Name != deploymentContainerName {
 .
.
.
}

Reply:
yes I changed it to reduce indetation as @qinqon asked [here](https://github.com/k8snetworkplumbingwg/kubemacpool/pull/152#discussion_r429901532)

Reply:
will move it back and function to also document this.